bindata/scripts/load-udev.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-echo "Reload udev rules: /usr/sbin/udevadm control --reload-rules"
-chroot /host/ /usr/sbin/udevadm control --reload-rules
+echo "Reload udev rules: udevadm control --reload-rules"
+chroot /host/ udevadm control --reload-rules
 
-echo "Trigger udev event: /usr/sbin/udevadm trigger --action add --attr-match subsystem=net"
-chroot /host/ /usr/sbin/udevadm trigger --action add --attr-match subsystem=net
+echo "Trigger udev event: udevadm trigger --action add --attr-match subsystem=net"
+chroot /host/ udevadm trigger --action add --attr-match subsystem=net
```

Comment:
it bad practice to remove the full path. We don't know how the PATH environment variable is setup. It is better to define the full path per OS

Reply:
updated